.github/workflows/OCV-Contrib-PR-3.4-W10.yaml

```diff
@@ -77,6 +78,27 @@ jobs:
         else
           echo "No merge since ${{ env.PR_AUTHOR }}/opencv_extra does not have branch ${{ env.SOURCE_BRANCH_NAME }}"
         fi
+    - name: Check DNN models update
+      shell: bash
+      working-directory: ${{ github.workspace }}
+      run: |
+        LATEST_SAVED_HASH=$(cat $DNN_MODELS\\dnn\\latest-hash-${{ env.TARGET_BRANCH_NAME }}.txt)
+        LATEST_HASH=$(sha256sum opencv_extra\\testdata\\dnn\\${{ env.DOWNLOAD_DNN_MODELS_FILE }} | awk '{print $1}')
+        if [[ "$LATEST_SAVED_HASH" == "$LATEST_HASH" ]]; then
+          echo "MATCHER=true" >> $GITHUB_ENV && MATCHER=true
```

Comment:
`echo "MATCHER=true" >> $GITHUB_ENV` is not remembered for the same step, that's why need to define one more time like this: `MATCHER=true`